apm-sniffer/apm-sdk-plugin/apm-armeria-plugins/apm-armeria-0.99.x-plugin/src/main/resources/skywalking-plugin.def

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+armeria-0.99.0=org.apache.skywalking.apm.plugin.armeria.define.Armeria099ClientInstrumentation
```

Comment:
This is a new plugin, `plugin-list.md` and `supported-list.md` should be updated.

Reply:
Plugin-list.md and supported-list.md have been updated